classes/classes/Output.as

```diff
@@ -12,7 +12,17 @@ package classes
 		import classes.GlobalFlags.kFLAGS;
 		import coc.view.MainView;
 
-		public function Output() {}
+		private static var _instance:Output = new Output();
+
+		public function Output()
```

Comment:
Don't know if this is different from what I know of Java, but ... technically when using "singleton" instances, isn't it better to make the constructor *private* in visibility? That is to say, the constructor should be hidden in visibility to anything outside the class, the static instance field is the only place this object should be constructed. (Unless you wish other classes to instantiate it, which nullifies the need for a singleton instance... My two coins of whatever denomination you choose.....)